promql/engine.go

```diff
@@ -352,12 +352,12 @@ func (ng *Engine) SetQueryLogger(l QueryLogger) {
 }
 
 // NewInstantQuery returns an evaluation query for the given expression at the given time.
-func (ng *Engine) NewInstantQuery(q storage.Queryable, qs string, ts time.Time) (Query, error) {
+func (ng *Engine) NewInstantQuery(q storage.Queryable, qs string, ts time.Time, lookbackDelta time.Duration) (Query, error) {
```

Comment:
Added a new parameter here, though I can do something else in order to keep the method backwards compatible.

Also, wasn't sure if it makes sense to add `lookbackDelta` parameter to `NewRangeQuery()`.

Reply:
`Also, wasn't sure if it makes sense to add lookbackDelta parameter to NewRangeQuery()`

Yes, this should applu to NewRangeQuery as well. Let's not make clients of the library update twice, let's do both at once.

Reply:
Since both `NewInstantQuery()` and `NewRangeQuery()` now accept `QueryOpts`, also both of them support custom lookback.

Though not sure how this affects range queries and if any tests could be written to validate that it works as expected.